release.sh

```diff
@@ -159,19 +159,10 @@ esac
 git push origin "${MVN_VERSION_RELEASE}"
 
 echo "";
-echo "One-jar build takes several minutes"
-read -n 1 -s -r -p "Press any key to continue to one-jar build (ctrl+c to cancel)"; printf "\n\n";
-
-# build one jar
-mvn -Passembly clean install -DskipTests
-
-echo "Starting utomated upload with sftp. Timeout is set to 1 hour!"
-
-./sftp-onejar-upload.expect $username $password $MVN_VERSION_RELEASE
-
-echo "";
-echo "Upload complete";
-echo "";
+echo "You need to tell jenkins to start the release process."
+echo "Go to: https://ci.eclipse.org/rdf4j/job/rdf4j-deploy-release-ossrh/ (if you are on linux or windows, remember to use CTRL+SHIFT+C)."
```

Comment:
"Remember to use Ctrl-Shift-C"? What's that for?

Reply:
Copy to clipboard. Since Linux and Windows use CTRL-C to copy everywhere except when you are in a terminal.